docusaurus/website/docusaurus.config.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @todo Setup prettier and what not... would be nice to use some of our own CLI to manage this
+ */
+const siteConfig = {
+  title: 'PixelOven',
+  tagline: 'A modern CLI for production ready JavaScript applications.',
+  url: 'https://pixeloven.com',
+  baseUrl: '/',
+  projectName: 'pixeloven',
+  organizationName: 'pixeloven',
+  favicon: 'img/favicon/favicon.ico',
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      {
+        docs: {
+          path: '../docs',
+          sidebarPath: require.resolve('./sidebars.json'),
+          editUrl:
+            'https://github.com/pixeloven/pixeloven/tree/master/docusaurus/docs',
+          showLastUpdateAuthor: true,
+          showLastUpdateTime: true,
+        },
+        theme: {
+          customCss: require.resolve('./src/css/custom.css'),
+        },
+      },
+    ],
+  ],
+  themeConfig: {
+    // image: 'img/logo-og.png', // What is this for ?
```

Comment:
Remove commented bits of the config

Reply:
Want to leave all these comments because I want to explore all this more and some of it isn't documented.